ui/apps/platform/src/services/CLIService.js

```diff
@@ -8,9 +8,9 @@ import { saveFile } from 'services/DownloadService';
 export default function downloadCLI(type) {
     let name = 'roxctl';
     let suffix = type;
-    if (type === 'windows') {
+    if (type === 'windows-amd64') {
```

Comment:
not your code, but given that who knows what archs we support in the future, I would prefer the more general
```
if (type.startsWith('windows-')) {
  name += 'exe';
  suffix += 'exe';
}
```
up to you